tools/sst_dump_test.cc

```diff
@@ -298,13 +298,74 @@ TEST_F(SSTDumpToolTest, NoSstFile) {
         "--command=verify", "--command=recompress", "--command=verify_checksum",
         "--show_properties"}) {
     snprintf(usage[1], kOptLength, "%s", command);
-    ASSERT_TRUE(!tool.Run(3, usage, opts));
+    ASSERT_TRUE(tool.Run(3, usage, opts));
   }
   for (int i = 0; i < 3; i++) {
     delete[] usage[i];
   }
 }
 
+TEST_F(SSTDumpToolTest, CheckValidSSTPath) {
+  Options opts;
+  opts.env = env();
+  std::string file_path = MakeFilePath("rocksdb_sst_test.sst");
+
+  char* usage[3];
+  PopulateCommandArgs(file_path, "--command=verify", usage);
+
+  SSTDumpTool tool;
+  // SST has not been created yet
+  ASSERT_TRUE(tool.Run(3, usage, opts));
+
+  // Create SST file
+  createSST(opts, file_path);
+  ASSERT_TRUE(!tool.Run(3, usage, opts));
+  cleanup(opts, file_path);
+
+  // Create text file
+  file_path = MakeFilePath("text_file");
+  std::string text = "Hello World!";
+  ASSERT_OK(WriteStringToFile(opts.env, text, file_path));
+  snprintf(usage[2], kOptLength, "--file=%s", file_path.c_str());
+  ASSERT_TRUE(tool.Run(3, usage, opts));
+  opts.env->DeleteFile(file_path);
+
+  for (int i = 0; i < 3; i++) {
+    delete[] usage[i];
+  }
+}
+
+TEST_F(SSTDumpToolTest, IdentifyValidSSTPath) {
+  Options opts;
+  opts.env = env();
+  std::string file_path = MakeFilePath("rocksdb_sst_test.sst");
+
+  char* usage[3];
+  PopulateCommandArgs(file_path, "--command=identify", usage);
+
+  SSTDumpTool tool;
+  // SST has not been created yet
+  ASSERT_TRUE(tool.Run(3, usage, opts));
+
+  // Create SST file
+  createSST(opts, file_path);
+  ASSERT_TRUE(!tool.Run(3, usage, opts));
+  cleanup(opts, file_path);
+
+  // Create text file
+  file_path = MakeFilePath("text_file");
+  std::string text = "Hello World!";
+  ASSERT_OK(WriteStringToFile(opts.env, text, file_path));
+  snprintf(usage[2], kOptLength, "--file=%s", file_path.c_str());
+  // Identify should exit successfully although the file is not SST
+  ASSERT_TRUE(!tool.Run(3, usage, opts));
```

Comment:
My intent with the identify command was to only return 0 if the file is a valid sst file. (I hadn't really thought about directories, but finding one seems sufficient to report success.)